wallet/src/Pos/Wallet/Web/Methods/Logic.hs

```diff
@@ -116,13 +117,15 @@ getAccountMod
     -> AccountId
     -> m CAccount
 getAccountMod balAndUtxo accMod accId = do
+    logInfo $ sformat ("getAccountMod: Account " % build % " has accMod: " % build) accId accMod
     dbAddrs    <- getAccountAddrsOrThrow Existing (NeedSorting True) accId
     let allAddrIds = gatherAddresses (camAddresses accMod) dbAddrs
```

Comment:
Might be useful to log the addresses in the account (or at least the _number_ of addresses)?